### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by rlmark**
… dependent libraries to installation section.

First stab at addressing #474.

### CODE REVIEW

**Comment by rlmark** [comment]
Thanks @L-Lavigne, I added the note about where the compiler plugin should go in the build.sbt file and that re-triggered the build. It looks like it passed!

File: docs/src/main/tut/index.md [hunk 1]
```diff
@@ -3,21 +3,33 @@ layout: home
...
 # Quickstart
 
-## What’s mu-rpc
+## What is mu-rpc?
 
 [mu] provides the ability to combine [RPC] protocols, services, and clients in your `Freestyle` program, thanks to [gRPC]. Although it's fully integrated with [gRPC], there are some important differences when defining the protocols, as we’ll see later on, since [mu] follows the same philosophy as `Freestyle` core, being macro-powered.
 
 ## Installation
 
+The current version for [mu] is `0.16.0` using the following common libraries and versions.
```
Comment (fedefernandez) [inline docs/src/main/tut/index.md]: With this comments, the version will be automatically updated on each release.

```suggestion

[comment]: # (Start Replace)

The current version for [mu] is `0.16.0` using the following common libraries and versions. 

[comment]: # (End Replace)
```

**Comment by fedefernandez** [comment]
I'll wait for this before continuing #472

**Comment by juanpedromoreno** [comment]
Talked to Fede in person, we can merge this now and include the Fede's suggestion in #472 ;)

File: docs/src/main/tut/index.md (hunk 1 above)
Comment (fedefernandez) [reply docs/src/main/tut/index.md]: Addressed in #472

**Pull request merged**